src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -3363,7 +3376,8 @@ pull_var_clause_default(Node *node)
 	 * PVC_REJECT_PLACEHOLDERS is implicit if PVC_INCLUDE_PLACEHOLDERS
 	 * isn't specified.
 	 */
-	List *columnList = pull_var_clause(node, PVC_RECURSE_AGGREGATES);
+	List *columnList = pull_var_clause(node, PVC_RECURSE_AGGREGATES |
```

Comment:
Additional flag is only there to prevent pull_var_clause function to fail.  Any aggregate function (or expression) window function contains already has to be inside target entry list. Therefore  going inside window function is not required.